packages/api/src/ApiClient.ts

```diff
@@ -45,6 +45,11 @@ export interface ApiConfig {
 	 */
 	authProvider: AuthProvider;
 
+	/**
+	 * Whether or not to use the new channel:manage:broadcast scope
+	 */
+	useNewBroadcastScope?: boolean;
```

Comment:
I would reverse the logic on this, i.e. name it `useLegacyBroadcastEditScope` and make it default to true.